priya/javascript/Assignment7/JsAssign7Q2.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bubble_sort</title>
+  </head>
+  <body>
+    <p style="text-align: center">
+      Number_of_student :<input type="text" id="num" placeholder="Enter number of students"><br>
+      <br>
+      <button onclick="bubble()">submit</button><br>
+      <br>
+    </p>
+    <script>
+      function bubble() {
```

Comment:
change function name and remove extra spaces in this file

Reply:
changes done